Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -229,7 +244,7 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
         do {
             switch message.type {
             case .onLoaded:
-                injectEditorConfiguration()
+                return
```

Comment:
I suggest we remove this now unused event by reverting the relevant portions of https://github.com/wordpress-mobile/GutenbergKit/pull/14/commits/3606c6dbea701cb03f534b62ca0c17a50e543ef6. WDYT?

Reply:
Sure! Removed in https://github.com/wordpress-mobile/GutenbergKit/pull/15/commits/5be8e6e592ac195256d880ab8d81fc305ff578e6